defaults/main.yml

```diff
@@ -21,6 +21,9 @@ nim_waku_log_level: 'info'
 # experimental discovery support
 nim_waku_discovery_enabled: false
 
+# DNS Discovery
+nim_waku_dns_disc_enabled: false
```

Comment:
We will need to have a commented out `nim_waku_dns_disc_url` here. Possibly with example value.

Reply:
Added in https://github.com/status-im/infra-role-nim-waku/pull/4/commits/fdaddbae2144de8faaafe8d2366f56e0b64db31c